gobblin-data-management/src/main/java/org/apache/gobblin/data/management/conversion/hive/source/HiveSource.java

```diff
@@ -27,6 +27,7 @@
 import lombok.extern.slf4j.Slf4j;
 
 import org.apache.commons.lang.StringUtils;
+import org.apache.gobblin.data.management.copy.hive.PartitionFilterGenerator;
```

Comment:
Please make sure you are following style guidance https://gobblin.readthedocs.io/en/latest/developer-guide/CodingStyle/

Reply:
Fixed. Thank you.